From b2929e314249bcf776aa79b260052e180fd57747 Mon Sep 17 00:00:00 2001
From: Tri Vo <trong@google.com>
Date: Mon, 13 May 2019 10:05:49 -0700
Subject: ANDROID: cuttlefish_defconfig: sync with
 android-mainline-tracking

Note that savedefconfig dropped CONFIG_SCHED_TUNE, which is not present
on this branch (yet).

Test: boot cuttlefish on arm64
Signed-off-by: Tri Vo <trong@google.com>
---
 arch/arm64/configs/cuttlefish_defconfig | 20 ++++++++------------
 1 file changed, 8 insertions(+), 12 deletions(-)

diff --git a/arch/arm64/configs/cuttlefish_defconfig b/arch/arm64/configs/cuttlefish_defconfig
index 435332a066621..5f7157425a316 100644
--- a/arch/arm64/configs/cuttlefish_defconfig
+++ b/arch/arm64/configs/cuttlefish_defconfig
@@ -14,11 +14,9 @@ CONFIG_MEMCG_SWAP=y
 CONFIG_RT_GROUP_SCHED=y
 CONFIG_CGROUP_FREEZER=y
 CONFIG_CPUSETS=y
-# CONFIG_PROC_PID_CPUSET is not set
 CONFIG_CGROUP_CPUACCT=y
 CONFIG_CGROUP_BPF=y
 CONFIG_SCHED_AUTOGROUP=y
-CONFIG_SCHED_TUNE=y
 CONFIG_BLK_DEV_INITRD=y
 # CONFIG_RD_BZIP2 is not set
 # CONFIG_RD_LZMA is not set
@@ -36,8 +34,7 @@ CONFIG_EMBEDDED=y
 # CONFIG_COMPAT_BRK is not set
 # CONFIG_SLAB_MERGE_DEFAULT is not set
 CONFIG_PROFILING=y
-CONFIG_PCI=y
-CONFIG_PCI_HOST_GENERIC=y
+CONFIG_SCHED_MC=y
 CONFIG_HZ_100=y
 CONFIG_SECCOMP=y
 CONFIG_PARAVIRT=y
@@ -47,8 +44,6 @@ CONFIG_CP15_BARRIER_EMULATION=y
 CONFIG_SETEND_EMULATION=y
 CONFIG_ARM64_SW_TTBR0_PAN=y
 CONFIG_RANDOMIZE_BASE=y
-CONFIG_CMDLINE="console=ttyAMA0"
-CONFIG_CMDLINE_EXTEND=y
 # CONFIG_EFI is not set
 CONFIG_COMPAT=y
 CONFIG_PM_WAKELOCKS=y
@@ -64,8 +59,6 @@ CONFIG_CPU_FREQ_DEFAULT_GOV_SCHEDUTIL=y
 CONFIG_CPU_FREQ_GOV_POWERSAVE=y
 CONFIG_CPU_FREQ_GOV_CONSERVATIVE=y
 CONFIG_CPUFREQ_DT=y
-CONFIG_ARM_BIG_LITTLE_CPUFREQ=y
-CONFIG_ARM_DT_BL_CPUFREQ=y
 CONFIG_ARM_SCPI_CPUFREQ=y
 CONFIG_ARM_SCMI_CPUFREQ=y
 CONFIG_ARM_SCMI_PROTOCOL=y
@@ -86,6 +79,8 @@ CONFIG_NET=y
 CONFIG_PACKET=y
 CONFIG_UNIX=y
 CONFIG_XFRM_USER=y
+CONFIG_XFRM_INTERFACE=y
+CONFIG_XFRM_STATISTICS=y
 CONFIG_NET_KEY=y
 CONFIG_INET=y
 CONFIG_IP_MULTICAST=y
@@ -196,6 +191,8 @@ CONFIG_CFG80211=y
 CONFIG_MAC80211=y
 # CONFIG_MAC80211_RC_MINSTREL is not set
 CONFIG_RFKILL=y
+CONFIG_PCI=y
+CONFIG_PCI_HOST_GENERIC=y
 # CONFIG_UEVENT_HELPER is not set
 CONFIG_DEVTMPFS=y
 # CONFIG_ALLOW_DEV_COREDUMP is not set
@@ -208,7 +205,6 @@ CONFIG_BLK_DEV_RAM_SIZE=8192
 CONFIG_VIRTIO_BLK=y
 CONFIG_UID_SYS_STATS=y
 CONFIG_SCSI=y
-# CONFIG_SCSI_MQ_DEFAULT is not set
 # CONFIG_SCSI_PROC_FS is not set
 CONFIG_BLK_DEV_SD=y
 CONFIG_SCSI_VIRTIO=y
@@ -285,6 +281,7 @@ CONFIG_SERIAL_8250_NR_UARTS=48
 CONFIG_SERIAL_8250_EXTENDED=y
 CONFIG_SERIAL_8250_MANY_PORTS=y
 CONFIG_SERIAL_8250_SHARE_IRQ=y
+CONFIG_SERIAL_OF_PLATFORM=y
 CONFIG_SERIAL_AMBA_PL011=y
 CONFIG_SERIAL_AMBA_PL011_CONSOLE=y
 CONFIG_VIRTIO_CONSOLE=y
@@ -384,6 +381,7 @@ CONFIG_MMC=y
 # CONFIG_MMC_BLOCK is not set
 CONFIG_RTC_CLASS=y
 # CONFIG_RTC_SYSTOHC is not set
+CONFIG_RTC_DRV_PL030=y
 CONFIG_RTC_DRV_PL031=y
 CONFIG_VIRTIO_PCI=y
 # CONFIG_VIRTIO_PCI_LEGACY is not set
@@ -401,13 +399,11 @@ CONFIG_MAILBOX=y
 # CONFIG_IOMMU_SUPPORT is not set
 CONFIG_ANDROID=y
 CONFIG_ANDROID_BINDER_IPC=y
-CONFIG_LEGACY_ENERGY_MODEL_DT=y
 CONFIG_EXT4_FS=y
 CONFIG_EXT4_FS_SECURITY=y
-CONFIG_EXT4_ENCRYPTION=y
 CONFIG_F2FS_FS=y
 CONFIG_F2FS_FS_SECURITY=y
-CONFIG_F2FS_FS_ENCRYPTION=y
+CONFIG_FS_ENCRYPTION=y
 # CONFIG_DNOTIFY is not set
 CONFIG_QUOTA=y
 CONFIG_QFMT_V2=y
-- 
2.23.0.444.g18eeb5a265-goog

